_posts/2020-04-16-status-update .md

```diff
@@ -0,0 +1,92 @@
+
+If you are as excited by all of this as I am then I urge you to read to blog post made by fellow team member [David Schmidt](david) found [here]({% link _posts/2020-03-30-debugging-unit-tests-with-honeycomb.md %}) and discover all that [Honeycomb](honeycomb) can do for you.
+
+## Module Releases
+
+One thing we are always happy to talk about is the release of a module, in this case:
+
+- [puppetlabs/stdlib](https://forge.puppet.com/puppetlabs/stdlib) (v6.3.0)
+
+## Continued Work on Puppet DSC
+
+The work done by the team to [refactor DSC support](iac-41) in Puppet has continued with strides being made every day and the first of the beta trial packets being sent out. To any communit members that have not recieved one of these packets but would like to, we encourage you to reach out to dsc@puppet.com with answers to the following questions:
+
+- What are the top three PowerShell modules with DSC resources you use/depend on the most?
+- Are you interested in participating in our beta testing program?
+
+## Community Issues
+
+Speaking of community I would like to take a moment to mention the community issues that have been raised over the last week and that are heping us to continually imrpove upon our modules and ensure that they run just as they should:
+
+- [MODULES-10628](https://tickets.puppetlabs.com/browse/MODULES-10628) - Docker volumes expects array rather than hash
+- [MODULES-10629](https://tickets.puppetlabs.com/browse/MODULES-10629) - Registry_auth results in changes every catalog run
+
+## Community Contributions
+
+When we talk of community it s not just the issues that they have raised, but also the contributions that thye have made, so here's a few of the excellent PR's opened by community members that have been merged in over the last week:
+
+- [Add ProxyIOBufferSize option #2014](https://github.com/puppetlabs/puppetlabs-apache/pull/2014)
+- [(MODULES-10623) explicitly top-scope calls to JSON methods #1101](https://github.com/puppetlabs/puppetlabs-stdlib/pull/1101)
+
+We would like to thank both [jplindquist](jplindquist) and [tkishel](tkishel) for these contributions and hope to see more from them both in the future.
+
+## Community Management
+
+In order to help us better work with the community our very own [Daiana](daiana) has continued to work on a management tool to help us keep track of the various PRs opened by the community with a new improvement being added just this week to help us better find which prs have last been given a responce, either from ourselves or by the cmmunity member that opened it.
```

Comment:
Would also suggest this could be broken up a bit. Some typos too:
```suggestion
In order to help us better work with the community our very own [Daiana](daiana) has continued to work on a management tool to help us keep track of the various PRs opened by the community.  A new improvement added this week was the ability for us to find out when PRs have last been given a response, either from ourselves or by the community member that opened it.
```